package.json

```diff
@@ -93,6 +94,11 @@
                 "title": "%containerApps.deployImage%",
                 "category": "Azure Container Apps"
             },
+            {
+                "command": "containerApps.deployImageApi",
+                "title": "%containerApps.deployImageApi%",
```

Comment:
Even though this command will not show in the palette or tree views, it could still be seen in the keyboard shortcuts editor, so it's best to give it _something_ for a title.